core/blockchain.go

```diff
@@ -1460,8 +1462,9 @@ func (bc *BlockChain) writeBlockWithState(block *types.Block, receipts []*types.
 		})
 
 		if len(blockLogs) > len(logs) {
-			rawdb.WriteBorReceipt(blockBatch, block.Hash(), block.NumberU64(), &types.BorReceiptForStorage{
-				Logs: blockLogs[len(logs):], // get state-sync logs from `state.Logs()`
+			rawdb.WriteBorReceipt(blockBatch, block.Hash(), block.NumberU64(), &types.ReceiptForStorage{
+				Status: types.ReceiptStatusSuccessful, // make receipt status successful
```

Comment:
Is `Status` something which denotes status of deposit/ state sync ?

Reply:
`Status` is about the tx status. In our case, it will be always successful as we are adding a new "fake" receipt for state-sync.